### DISCUSSION TITLE
使用ant-design时 能不能完全避免使用: global 来覆盖一些样式

### DISCUSSION THREAD

**Discussion by kelisiWu123** [ant-design/ant-design#51274] [Q&A] [answered]
使用ant-design时 能不能完全避免使用 :global 来覆盖一些样式

**Comment by zombieJ** [comment]
antd 没用过 `:global`，请检查一下你使用的框架是否做了一些额外的定制。

**Reply by kelisiWu123** [reply]
不好意思，是我没有表达清楚，我想问的是在使用antd时，例如我想修改Modal的title的margin时等此类操作时，是否有比使用 :global 更好的方式

**Reply by yoyo837** [reply]
token

**Reply by kelisiWu123** [reply]
token 确实是能定制，但是太细的好像不可以

**Reply by afc163** [reply]
比如哪些需求，可以按需开 token。

**Reply by kelisiWu123** [reply]
比方说我只想在某个页面内，改变Modal组件的标题的下边距，是不是避免不了 通过例如`.container{ :global . ant-modal-title{ marginBottom: 20} }` 这样的方式进行修改

**Reply by afc163** [reply, marked as answer]
```jsx
<Modal styles={{ header: { marginBottom: 24 } }} />
```

试试这样？
